bonfire/Modules/Tools/Controllers/LogsController.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+namespace Bonfire\Tools\Controllers;
+
+use App\Controllers\AdminController;
+use CodeIgniter\CodeIgniter;
+
+class LogsController extends AdminController
+{
+    protected $theme = 'Admin';
+
+    protected $viewPrefix = 'Bonfire\Modules\Tools\Views\\';
+
+    protected $logs_path = WRITEPATH . 'logs/';
```

Comment:
Please use camelCase for variable names.

```suggestion
    protected $logsPath = WRITEPATH . 'logs/';
```